js/front/dataset/resource-modal.vue

```diff
@@ -10,20 +10,16 @@
           <dd><a :href="resource.url" @click="onClick">{{resource.url}}</a></dd>
           <dt v-if="resource.encodingFormat">{{ _('Format') }}</dt>
           <dd v-if="resource.encodingFormat">{{resource.encodingFormat}}</dd>
-          <dt v-if="resource.mimetype">{{ _('MimeType') }}</dt>
-          <dd v-if="resource.mimetype">{{resource.mimetype}}</dd>
           <dt v-if="resource.contentSize">{{ _('Size') }}</dt>
           <dd v-if="resource.contentSize">{{ resource.contentSize|size }}</dd>
           <dt v-if="resource.checksum">{{ resource.checksumType || 'sha1'}}</dt>
-          <dd v-if="resource.checksum">{{checksum}}</dd>
+          <dd v-if="resource.checksum">{{ resource.checksum }}</dd>
           <dt v-if="resource.dateCreated">{{ _('Created on') }}</dt>
           <dd v-if="resource.dateCreated"> {{ resource.dateCreated|dt }}</dd>
           <dt v-if="resource.dateModified">{{ _('Modified on') }}</dt>
           <dd v-if="resource.dateModified"> {{ resource.dateModified|dt }}</dd>
           <dt v-if="resource.datePublished">{{ _('Published on') }}</dt>
           <dd v-if="resource.datePublished"> {{ resource.datePublished|dt }}</dd>
-          <dt v-if="resource.downloads">{{ _('Downloads') }}</dt>
-          <dd v-if="resource.downloads"> {{ resource.downloads }}</dd>
```

Comment:
It should be `{{ resource.interactionStatistic.userInteractionCount }}` as provided in the JSON-LD, if present